tests/schemas/api_v3/tokenIDs.schema.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import Joi from 'joi';
+
+const tokenIDsMetaSchema = {
+	count: Joi.number().integer().min(0).required(),
+	offset: Joi.number().integer().min(0).required(),
+	total: Joi.number().integer().min(0).required(),
+};
+
+const tokenIDsSchema = {
+	tokenIDs: Joi.array().required(),
+};
+
+const goodRequestSchemaFortokenIDs = {
```

Comment:
Previously, the container schema used to be named `goodRequestSchema`, then we started naming in respect to the api name (ex: tokenIDsResponseSchema in this case). Maybe we can do either of below or something similar

```suggestion
const goodResponseSchemaFortokenIDs = {
```
OR

```suggestion
const tokenIDsResponseSchema = {
```